### PR TITLE
focusable select box

### PR DESCRIPTION
**Pull request by Neill83**
@plentymarkets/team-terra

### Definition of Done
Documentation
- [x] Changelog

Testing
- [x] Unit Test successful
- [x] Person 1
- [x] Person 2

Browser-Support
- [x] Chrome
- [x] Firefox
- [x] Safari

### CODE REVIEW

File: src/app/components/forms/select-box/terra-select-box.component.html [hunk 1]
```diff
@@ -10,7 +10,8 @@
                   'isEmpty': emptyValueSelected
                 }"
      (click)="onClick($event)"
-     container="body">
+     container="body"
+     tabindex="0">
```
Comment (pweyrich) [inline src/app/components/forms/select-box/terra-select-box.component.html]: Ist das tatsächlich noch notwendig?

Comment (Neill83) [reply src/app/components/forms/select-box/terra-select-box.component.html]: ja

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+###unreleased
```
Comment (pweyrich) [inline CHANGELOG.md]:
```suggestion
# Unreleased
```

**Comment by pweyrich** [comment]
bildschirmfoto 2018-12-05 um 10 44 27
Unit tests successful 👍

**Pull request merged**